lib/action_controller/serialization.rb

```diff
@@ -25,7 +25,8 @@ def get_serializer(resource, options = {})
           "Please pass 'adapter: false' or see ActiveSupport::SerializableResource.new"
         options[:adapter] = false
       end
-      serializable_resource = ActiveModel::SerializableResource.new(resource, options)
+      serializable_options = options.merge(controller: self)
```

Comment:
Currently we just need the class name but there's virtually 0 overhead to getting the instance, so why not?